--- a/src/ffmpeg.js
+++ b/src/ffmpeg.js
@@ -21,17 +21,7 @@
 }
 
 function getFfPath(cmd) {
-  // Testing non-mac setup on mac:
-  // return `node_modules/ffmpeg-ffprobe-static/${cmd}`;
-
-  if (isMac) {
-    return isDev ? `ffmpeg-mac/${cmd}` : join(window.process.resourcesPath, cmd);
-  }
-
-  const exeName = isWindows ? `${cmd}.exe` : cmd;
-  return isDev
-    ? `node_modules/ffmpeg-ffprobe-static/${exeName}`
-    : join(window.process.resourcesPath, `node_modules/ffmpeg-ffprobe-static/${exeName}`);
+  return cmd;
 }
 
 export const getFfmpegPath = () => getFfPath('ffmpeg');
