--- a/src/ffmpeg.js
+++ b/src/ffmpeg.js
@@ -26,16 +26,7 @@
 }
 
 function getFfPath(cmd) {
-  const platform = os.platform();
-
-  if (platform === 'darwin') {
-    return isDev ? `ffmpeg-mac/${cmd}` : join(window.process.resourcesPath, cmd);
-  }
-
-  const exeName = platform === 'win32' ? `${cmd}.exe` : cmd;
-  return isDev
-    ? `node_modules/ffmpeg-ffprobe-static/${exeName}`
-    : join(window.process.resourcesPath, `node_modules/ffmpeg-ffprobe-static/${exeName}`);
+  return cmd;
 }
 
 const getFfmpegPath = () => getFfPath('ffmpeg');
