--- a/public/electron.js
+++ b/public/electron.js
@@ -34,6 +34,7 @@
       nodeIntegration: true,
       // https://github.com/electron/electron/issues/5107
       webSecurity: !isDev,
+      enableRemoteModule: true
     },
   });
 
@@ -110,7 +111,7 @@
 
 electron.ipcMain.on('renderer-ready', () => {
   rendererReady = true;
-  const ignoreFirstArgs = isDev ? 2 : 1;
+  const ignoreFirstArgs = 2;
   // production: First arg is the LosslessCut executable
   // dev: First 2 args are electron and the electron.js
 
