--- a/src/ffmpeg.js
+++ b/src/ffmpeg.js
@@ -5,14 +5,12 @@ import moment from 'moment';
 import i18n from 'i18next';
 import Timecode from 'smpte-timecode';
 
-import { getOutPath, isDurationValid, getExtensionForFormat, isMac, isWindows } from './util';
+import { getOutPath, isDurationValid, getExtensionForFormat } from './util';
 
 const execa = window.require('execa');
-const { join } = window.require('path');
 const fileType = window.require('file-type');
 const readChunk = window.require('read-chunk');
 const readline = window.require('readline');
-const isDev = window.require('electron-is-dev');
 
 
 export function getFfCommandLine(cmd, args) {
@@ -21,17 +19,7 @@ export function getFfCommandLine(cmd, args) {
 }
 
 function getFfPath(cmd) {
-  // Testing non-mac setup on mac:
-  // return `node_modules/ffmpeg-ffprobe-static/${cmd}`;
-
-  if (isMac) {
-    return isDev ? `ffmpeg-mac/${cmd}` : join(window.process.resourcesPath, cmd);
-  }
-
-  const exeName = isWindows ? `${cmd}.exe` : cmd;
-  return isDev
-    ? `node_modules/ffmpeg-ffprobe-static/${exeName}`
-    : join(window.process.resourcesPath, `node_modules/ffmpeg-ffprobe-static/${exeName}`);
+  return cmd;
 }
 
 export const getFfmpegPath = () => getFfPath('ffmpeg');
