--- a/src/Utils/DockManager.vala
+++ b/src/Utils/DockManager.vala
@@ -30,23 +30,9 @@
     }
 
     private void progress_dock_visibility (bool visible) {
-        Granite.Services.Application.set_progress_visible.begin (visible, (obj, res) => {
-            try {
-                Granite.Services.Application.set_progress_visible.end (res);
-            } catch (GLib.Error e) {
-                critical (e.message);
-            }
-        });
     }
 
     private void update_dock_progress (double progress) {
-        Granite.Services.Application.set_progress.begin (progress, (obj, res) => {
-            try {
-                Granite.Services.Application.set_progress.end (res);
-            } catch (GLib.Error e) {
-                GLib.critical (e.message);
-            }
-        });
     }
 }
-    
\ 文件尾没有换行符
+    
