--- a/src/Widgets/HeaderBar.vala
+++ b/src/Widgets/HeaderBar.vala
@@ -115,10 +115,9 @@
             });
 
 
-            Gtk.Image multiple_icon = new Gtk.Image () {
-                gicon = new ThemedIcon ("focus-legacy-systray-symbolic.symbolic"),
-                icon_size = Gtk.IconSize.LARGE_TOOLBAR
-            };
+            Gtk.Image multiple_icon = new Gtk.Image ();
+            multiple_icon.gicon = new ThemedIcon ("focus-legacy-systray-symbolic.symbolic");
+            multiple_icon.icon_size = Gtk.IconSize.LARGE_TOOLBAR;
 
             multiple_menu = new Gtk.Button();
             var popup_multiple = new MultipleWallpaperPopover (multiple_menu, multiple_wallpaper);
