--- a/src/Layouts/HeaderBar.vala
+++ b/src/Layouts/HeaderBar.vala
@@ -452,9 +452,7 @@
                         break;
                 }
 
-                button.get_child ().destroy ();
-                var label = new Granite.AccelLabel.from_action_name (file_name, accels);
-                button.add (label);
+                button.text = file_name;
                 button.action_name = accels;
             } else {
                 button.text = file_name;
@@ -542,20 +540,13 @@
 
     private Gtk.ModelButton create_model_button (string text, string? icon, string? accels = null) {
         var button = new Gtk.ModelButton ();
-        button.get_child ().destroy ();
-        var label = new Granite.AccelLabel.from_action_name (text, accels);
+        button.text = text;
 
         if (icon != null) {
             var image = new Gtk.Image.from_icon_name (icon, Gtk.IconSize.MENU);
-            image.margin_end = 6;
-            label.attach_next_to (
-                image,
-                label.get_child_at (0, 0),
-                Gtk.PositionType.LEFT
-            );
+            button.image = image;
         }
 
-        button.add (label);
         button.action_name = accels;
 
         return button;
