--- a/src/MainWindow.vala
+++ b/src/MainWindow.vala
@@ -379,44 +379,12 @@
 
         public void add_badge(){
             count_badge += 1;
-            Granite.Services.Application.set_badge_visible.begin (true, (obj, res) => {
-                try {
-                    Granite.Services.Application.set_badge_visible.end (res);
-                } catch (GLib.Error e) {
-                    critical (e.message);
-                }
-            });
-            Granite.Services.Application.set_badge.begin (count_badge, (obj, res) => {
-                try {
-                    Granite.Services.Application.set_badge.end (res);
-                } catch (GLib.Error e) {
-                    critical (e.message);
-                }
-            });
         }
         public void remove_badge(){
             if(count_badge <= 0){
                 return;
             }
             count_badge -= 1;
-            Granite.Services.Application.set_badge.begin (count_badge, (obj, res) => {
-                try {
-                    Granite.Services.Application.set_badge.end (res);
-                } catch (GLib.Error e) {
-                    critical (e.message);
-                }
-            });
-            
-            if(count_badge == 0){
-                Granite.Services.Application.set_badge_visible.begin (false, (obj, res) => {
-                    try {
-                        Granite.Services.Application.set_badge_visible.end (res);
-                    } catch (GLib.Error e) {
-                        critical (e.message);
-                    }
-                });
-            }
-            
         }
         public void action_edit_conn (string name) {
             sourcelist.edit_conn(name);
