--- a/src/chord/code/electron-main/main.ts
+++ b/src/chord/code/electron-main/main.ts
@@ -62,6 +62,7 @@
         webPreferences: {
             nodeIntegration: true,
             enableRemoteModule: true,
+            contextIsolation: false
         },
     });
 
--- a/src/chord/music/xiami/webApi.ts
+++ b/src/chord/music/xiami/webApi.ts
@@ -172,6 +172,7 @@
                 webPreferences: {
                     enableRemoteModule: false,
                     disableHtmlFullscreenWindowResize: true,
+                    contextIsolation: false
                 },
             });
         // win.webContents.openDevTools();
--- a/src/chord/music/qq/api.ts
+++ b/src/chord/music/qq/api.ts
@@ -1042,7 +1042,8 @@
                 webPreferences: {
                     webSecurity: true,
                     nodeIntegration: false,
-                    allowRunningInsecureContent: true
+                    allowRunningInsecureContent: true,
+                    contextIsolation: false
                 },
             });
         // win.webContents.openDevTools();
